ztp/siteconfig-generator/siteConfig/siteConfig.go

```diff
@@ -232,3 +243,8 @@ type Interfaces struct {
 	Name       string `yaml:"name"`
 	MacAddress string `yaml:"macAddress"`
 }
+
+// BiosConfigMapRef
+type BiosConfigMapRef struct {
+	Name string `yaml:"name"`
```

Comment:
As we discussed , we better let the user enter the configMap name and add another field for the filePath.